Apps/Playground/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ set(BABYLONSCRIPTS
     "../BabylonScripts/babylon.max.js"
     "../BabylonScripts/babylonjs.materials.js"
     "../BabylonScripts/ammo.js"
+    "../BabylonScripts/meshwriter.min.js"
```

Comment:
So this is not part of Babylon.js itself, correct? It *looks* like in the React Native case we would just install the [meshwriter NPM package](https://www.npmjs.com/package/meshwriter), does that sound right?

Reply:
That sounds right to me. I haven't tried it, but I just copied this file out of the Babylon.js extensions repository, so it presumably should be the same as the one on NPM.